src/main/java/appeng/tile/misc/TileInscriber.java

```diff
@@ -281,25 +281,23 @@ private boolean hasWork()
 	@Nullable
 	public IInscriberRecipe getTask()
 	{
-		return getTask( this.sideItemHandler.getStackInSlot( 0 ), this.topItemHandler.getStackInSlot( 0 ), this.bottomItemHandler.getStackInSlot( 0 ) );
+		return this.getTask( this.sideItemHandler.getStackInSlot( 0 ), this.topItemHandler.getStackInSlot( 0 ), this.bottomItemHandler.getStackInSlot( 0 ) );
 	}
 
 	@Nullable
 	private IInscriberRecipe getTask( final ItemStack input, final ItemStack plateA, final ItemStack plateB )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "getTask" is 28 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)